airbyte-ci/connectors/pipelines/pipelines/hacks.py

```diff
@@ -86,7 +86,7 @@ def do_regression_test_status_check_maybe(ctx: click.Context, status_check_name:
     run_url = ctx.obj["gha_workflow_run_url"]
     should_send = ctx.obj.get("ci_context") == consts.CIContext.PULL_REQUEST
 
-    if any(
+    if is_automerge_pull_request(ctx.obj["pull_request"]) and any(
```

Comment:
Don't we want the opposite behavior: when the pr has the automerge label we want to make the status check pass?